pom.xml

```diff
@@ -369,7 +374,7 @@
     </distributionManagement>
     <properties>
         <carbon.identity.framework.version>7.7.140</carbon.identity.framework.version>
-        <carbon.identity.oauth.version>7.0.286</carbon.identity.oauth.version>
+        <carbon.identity.oauth.version>7.0.323-SNAPSHOT</carbon.identity.oauth.version>
```

Comment:
Incorrect version

Reply:
Once https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2843 is merged, will update the corresponding released version.